src/digitalmarketplace/components/option-select/option-select.yaml

```diff
@@ -0,0 +1,96 @@
+params:
+- name: title
+  type: string
+  required: true
+  description: The title of the option select
+- name: name
+  type: string
+  required: true
+  description: The name of the option select element
+- name: options_container_id
+  type: string
+  required: true
+  description: The ID of the container for the Checkboxes component
```

Comment:
Why is this needed?

Reply:
Finder uses it for querying and because they have the searchbox in the same parent container. 

DMp has it in place on the old option boxes, and I figured it provides an easy route to the checkboxes component, and there might be some obscure use of the ID somewhere in a backend function that I'd mess up if I removed it!